vcd/datasource_vcd_org_test.go

```diff
@@ -102,7 +102,7 @@ resource "vcd_org" "{{.OrgName2}}" {
   delay_after_power_on_seconds = data.vcd_org.{{.OrgName1}}.delay_after_power_on_seconds
   delete_force                 = "true"
   delete_recursive             = "true"
- vapp_lease {
+  vapp_lease {
```

Comment:
`metadata` testing part missing

Reply:
Good one! I added another step in the test. The step fetches the organization created in previous step, which contains metadata.
As this new step depends on previous Terraform generated file, I've excluded it from binary tests.